### PR TITLE
add support for testnet

### PR DESCRIPTION
**Pull request by Moejoe90**

### CODE REVIEW

File: src/web3/utils.js [hunk 1]
```diff
@@ -16,3 +16,18 @@ export const validateBlockNumber = (lastBlockNumber, blockNumber) => {
...
+const getEtherNetworkId = () => web3Instance.version.network;
+
+const web3Utils =
+{
+  isAddress,
+  validateBlockNumber,
+  getEtherNetworkId,
+};
+
+export default web3Utils;
```
Comment (MostafaBalata) [inline src/web3/utils.js]: why not just export each function separately?

File: src/etherscan.js [hunk 2]
```diff
@@ -32,8 +33,16 @@ const validateResponse = (response) => {
  */
 
 const scrapeABI = async (address) => {
+  const networkID = web3Utils.getEtherNetworkId();
+  const networks = {
+    1: 'api',
+    3: 'ropsten',
+    4: 'rinkeby',
+    42: 'kovan',
```
Comment (MostafaBalata) [inline src/etherscan.js]: Isn't better to put this config into config file?

File: src/web3/utils.js (hunk 1 above)
Comment (Moejoe90) [reply src/web3/utils.js]: By design 
sinon doesn't stub methods without them being wrapped in an Object thttps://github.com/sinonjs/sinon/issues/562

The other solutions provided are very dirty honestly. So, we either keep it like this or we make etherscan module take an extra networkID param and pass through to this method.

File: src/etherscan.js (hunk 2 above)
Comment (Moejoe90) [reply src/etherscan.js]: How do you mean? 
Do you mean as an ENV var?

Comment (MostafaBalata) [reply src/etherscan.js]: No I meant in `config.js`, just put the object in there.

Comment (Moejoe90) [reply src/etherscan.js]: good idea

**Pull request merged**